src/main/java/core/basesyntax/Ball.java

```diff
@@ -0,0 +1,20 @@
+package core.basesyntax;
+
+public class Ball {
+    private String color;
+    private int number;
+
+    public Ball(String color, int number) {
+        this.color = color;
+        this.number = number;
+    }
+
+    public Ball(Color randomColor, int randomNumber) {
```

Comment:
delete this constructor

Reply:
not fixed.
constructor should not contain this logic `this.color = randomColor.name();`